packages/main/src/GridCellBase.ts

```diff
@@ -31,6 +31,9 @@ abstract class GridCellBase extends UI5Element {
 	@property({ type: Boolean })
 	_popin!: boolean;
 
+	@property({ type: String, defaultValue: "Left" })
```

Comment:
type cannot be String but Enum

Reply:
See main/src/types/TableOverflowMode.ts as an example, if you need one

Reply:
this line needs to be adapted after a rebase from main according to #8846